### DISCUSSION TITLE
Flash tuya ts0115

### DISCUSSION THREAD

**Discussion by vyalekseev** [arendst/Tasmota#12102] [Q&A] [answered]
Hello everyone! Can you please tell me if it is possible to flash Tuya ts0115 with tasmota firmware?
https://www.zigbee2mqtt.io/devices/TS0115.html

**Comment by sfromis** [comment, marked as answer]
No Zigbee devices can be flashed with Tasmota firmware. However, you'd likely be able to use it with a Zigbee [coordinator](https://zigbee.blakadder.com/coordinators.html) flashed with Tasmota, and thus get full local control. One popular option is the Sonoff ZbBridge. However, when it comes to Tuya, they don't tend to just follow standards, but I'd not expect too much trouble with such a simple device.
